Makefile

```diff
@@ -97,7 +97,8 @@ else
 endif
 
 behat-tests-v2-goutte-parallel: up-app-integration-tests reset-fixtures disable-debug ##@behat Run the integration tests in parallel
-	docker-compose -f docker-compose.yml -f docker-compose.dev.yml run --rm test sh ./tests/Behat/run-tests-parallel.sh --tags @v2
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml run --rm test sh ./tests/Behat/run-tests.sh --tags @v2_sequential
+	docker-compose -f docker-compose.yml -f docker-compose.dev.yml run --rm test sh ./tests/Behat/run-tests-parallel.sh --tags @v2&&~@v2_sequential
```

Comment:
`@v2&&~@v2_sequential`

Not sure if this is a mistype or an actual operator. Wanted to raise it just in case

Reply:
Yeah totally intentional - I'm saying here to run all tests with a v2 tag but to skip any that have v2_sequential (as we want them to run in sequence). This puts us more in line with how our tests run in CI which is nice.